olexandr.vakulinskiy/Task2/hash.rb

```diff
@@ -0,0 +1,16 @@
+p hash = {
+          'Yes' => 23,
+          'b' => 'travel',
+          'yesterday' => 34,
+           5 => '234',
+           yesss: :fg,
+           try: 30,
+           key: 'some value',
+          ' yesterday1 '=> 34,
+          ' yesteryear '=> 2014
+}
+
+keys = hash.transform_keys{|k| k.to_s}.keys
+
+puts "\nAmount of keys, which start with \"yes\""
+p keys.map{|key|  key.downcase.match(/^yes/)}.compact.size
```

Comment:
this will not find keys that start with a space, like this: `' yesterday1 '=> 34,`

Reply:
But in the slack was said that program must be space sensitive